qa/run_sc_tests.py

```diff
@@ -40,15 +40,15 @@ def run_tests(log_file):
     result = run_test(MCSCNodesAlive())
     assert_equal(0, result, "mc_sc_nodes_alive test failed!")
 
-    result = run_test(SCBackwardTransfer())
-    assert_equal(0, result, "sc_backward_transfer test failed!")
-
     result = run_test(SCBootstrap())
```

Comment:
`MCSCNodesAlive` and `SCBootstrap` tests should be executed before `MCSCForging1`